tsconfig.json

```diff
@@ -1,12 +1,23 @@
 {
   "compilerOptions": {
     "outDir": "./static/js/dist",
-    "noImplicitAny": true,
-    "module": "es6",
-    "target": "es5",
-    "jsx": "react",
+    "baseUrl": "src",
+    "target": "ES2020",
+    "module": "esnext",
+    "jsx": "react-jsx",
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+    "forceConsistentCasingInFileNames": true,
+    "lib": ["dom", "dom.iterable", "esnext"],
     "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "noFallthroughCasesInSwitch": true,
     "moduleResolution": "node",
-    "strict": true
-  }
+    "resolveJsonModule": true,
+    "isolatedModules": true,
+    "noEmit": false,
+    "sourceMap": true
+  },
+  "include": ["./static/js/brand-store/NoStores"]
```

Comment:
You may now be able to add just `./static/js` and have it work without specifying each folder manually.

Reply:
Yes, it does! Thanks, updated